.github/workflows/docker.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        version: ["0.5", "0.6"]
+        version: ["0.5", "0.6", "1.0"]
```

Comment:
Looks like this doesn't work for the Trivy vulnerability scanner step as that tries to scan an image with the tag `1.0` which isn't available as this currently only uses the tag `1.0.0-rc1` for this release candidate.
Not sure yet how to solve this.

Reply:
I re-enabled all tags also for non-stable versions like 1.0.0-rc1, but these tags are not pushed. This lets the security scanner work with the new image.
I've pushed that as a separate commit because I'm not sure if we really want to do this. If we want to do this, then I can squash the two commits, if someone has a better suggestion, then I can revert / modify it.